frontend/apps/design-system-storybook/package.json

```diff
@@ -5,10 +5,12 @@
     "code.org"
   ],
   "scripts": {
+    "clean": "rimraf storybook-static",
```

Comment:
is `rimraf` expected to be installed globally?

Reply:
No, it is locally installed as a npm package and will be made available when `yarn install` is done.